chess/svg.py

```diff
@@ -80,7 +88,7 @@ def piece(piece):
     return _svg(PIECES[piece.symbol()], 45, 45)
 
 
-def board(board=None, squares=None, flipped=False, coordinates=True, lastmove=None, check=None, size=400, style=None, pre="", post=""):
+def board(board=None, squares=None, flipped=False, coordinates=True, lastmove=None, check=None, size=400, style=None, pre="", post="", arrows=[]):
```

Comment:
Lets have this after `check` and before `size`. (Technically that would break backwards compability, but these are basically keyword-only arguments).